src/webui/www/private/downloadlimit.html

```diff
@@ -4,11 +4,11 @@
 <head>
     <meta charset="UTF-8" />
     <title>QBT_TR(Torrent Download Speed Limiting)QBT_TR[CONTEXT=TransferListWidget]</title>
-    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <link rel="stylesheet" href="css/style.css?v=${VERSION}" type="text/css" />
     <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
     <script src="scripts/lib/mootools-1.2-more.js"></script>
-    <script src="scripts/lib/mocha-yc.js"></script>
-    <script src="scripts/lib/parametrics.js"></script>
+    <script src="scripts/lib/mocha-0.9.6-yc.js"></script>
+    <script src="scripts/parametrics.js?v=${VERSION}"></script>
```

Comment:
Isn't it 3rd party code? Why did you move it outside "lib"?

Reply:
It looks like it was a third party script that we heavily modified.

Reply:
@Chocobo1, can you comment?

Reply:
The file is full of qbt specific code, maybe it should be merged into other js file, IMO it is optional for this PR.

Reply:
It's only used by `uploadlimit.html` and `downloadlimit.html`, which don't load any other first party js. I'd like to keep this as a separate file, but maybe we can rename it?

Reply:
>I'd like to keep this as a separate file, but maybe we can rename it?

Sure!

Reply:
Renamed file to `speedslider.js`